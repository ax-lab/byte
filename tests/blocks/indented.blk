let expr_1 =
	1 +
	2 +
	3

let expr_2
	= 4
	+ 5
	+ 6

let expr_3 = 7 + 8
	+ 9 + 10
expr_1 = expr_2 = expr_3
	a01
	a02
		b03
		b04
	a05
		b06
			c07
				d08
		b09
	a10
	a11
end
