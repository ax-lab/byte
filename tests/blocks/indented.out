Line(
	let expr_1 =
	...Line(
		1 +
	)
	...Line(
		2 +
	)
	...Line(
		3
	)
)
Line(
	let expr_2
	...Line(
		= 4
	)
	...Line(
		+ 5
	)
	...Line(
		+ 6
	)
)
Line(
	let expr_3 = 7 + 8
	...Line(
		+ 9 + 10
	)
)
Line(
	expr_1 = expr_2 = expr_3
	...Line(
		a01
	)
	...Line(
		a02
		...Line(
			b03
		)
		...Line(
			b04
		)
	)
	...Line(
		a05
		...Line(
			b06
			...Line(
				c07
				...Line(
					d08
				)
			)
		)
		...Line(
			b09
		)
	)
	...Line(
		a10
	)
	...Line(
		a11
	)
)
Line(
	end
)
